src/main/java/app/fitbuddy/controller/crud/HistoryController.java

```diff
@@ -102,6 +102,6 @@ public void delete(@PathVariable("id") Integer historyId) {
 					throw new FitBuddyException("UserIds doesn't match. Cannot delete others History.");
 				}
 			}
-		}	
+		
```

Comment:
- Remove empty line in Line 94
- Fix indentation in Line 95 to 104
- Remove empty line in Line 105